examples/apeworx_everlasting_bots.py

```diff
@@ -336,7 +338,7 @@ def get_agents(config):
     sol_agents = {"governance": governance}
     for agent_address, sim_agent in enumerate(sim_agents):
         sol_agent = ape.accounts.test_accounts.generate_test_account()  # make a fake agent with its own wallet
-        sol_agent.balance = to_fixed_point(sim_agent.budget)
+        sol_agent._set_balance_(to_fixed_point(sim_agent.budget))  # pylint: disable=protected-access
```

Comment:
might as well make a method that is public

Reply:
ya cus without a setter, we get this complaint: `Cannot assign member "balance" for type "TestAccountAPI"
  "int" is incompatible with "property"`

Reply:
switched to the public `set_balance(address, amount)` function in the `provider` we can access directly through `sol_agent.provider`